cygwin.rst

```diff
@@ -0,0 +1,194 @@
+*****************************
+Docs in Windows (Cygwin) 
+*****************************
+
+.. _cygwin-docs:
+
+Cygwin
+-------------------------------
+Cygwin is windows implementation of unix-like commands. You can basically use many linux commands from windows 
+command propmpt.
+
+Install `cygwin <https://cygwin.com/install.html/>`_ and add its path into windows environmental variable.
+This will allow you to `use cygwin from cmd promopt <https://www.howtogeek.com/howto/41382/how-to-use-linux-commands-in-windows-with-cygwin/>`_.
+
+
+.. _python:
+
+Python 3
+-------------------------------
+You need to install `python 3 <https://www.python.org/downloads/>`_.
+During the installation, you can select to set up pip tool as well as add python path to the Path environment variable.
+
+
+
+.. _virtualenv:
+
+Virtual Environment
+-------------------------------
+Virtual environment is a useful tool particularly if you work with multiple projects, each requires different dependencies and python versions. By using virtualenv, you can create seperated virtual environment and install packages as you need without affecting the main 
+python installation.
+
+To install virtualenv in windows, you can basically use pip command, which was already installed with python 3.6.2.
+
+.. code-block:: rest
```

Comment:
terminal commands are not `rest` (restructured text)
they should be `none` 

(If I had some command prompt samples with `rest`, it is a mistake we need to fix.

Reply:
Ok, got it!. Thanks for this info.

Reply:
Is it "rest" or " rst"?

Reply:
.rst is the file extension

Sphinx documentation seems to prefer `rest` for code-blocks for some reason. I think either actually work just fine.